epoch/api/README.md

```diff
@@ -3,26 +3,29 @@
 This document:
 * Provides an overview of the API exposed by the epoch node;
 * Defines the WebSocket API of the epoch node;
-* Describes the intented usage of the user API of the epoch node.
+* Describes the intended usage of the user API of the epoch node.
 
 ## Overview
 
-The epoch node API consists of the following endpoints:
-* External HTTP API;
-  * It consists of the peer-to-peer network API and of part of the user API;
-  * It is defined via Swagger schema;
+The epoch node exposes the following APIs:
+* Peer-to-peer network API. It consists of one TCP endpoint:
+  * It is encrypted and authenticated using the Noise protocol;
+  * It is not defined in a document. Advanced users may inspect the `aec_peer_connection` module;
```

Comment:
not yet? (we intend to describe, just not in this intermediate state)

Reply:
Addressed.